docker-hub/roles-and-permissions.md

```diff
@@ -0,0 +1,72 @@
+---
+description: Use roles in your organization to control who has access to content, registry, and organization management permissions.
+keywords: members, teams, organization, company, roles, access
+title: Roles and permissions
+---
+
+Organization and company owners can assign roles to individuals giving them different permissions in the organization. This section is for owners who want to learn about the defined roles and their permission scopes.
+
+## Roles
+
+When you invite users to your organization, you assign a role. A role is a collection of permissions. Roles define access to perform actions like creating repositories, pulling images, creating teams, and configuring organization settings.
+
+The following roles are available to assign:
+
+- **Member** - Non-administrative role. Members can view other members that are in the same organization.
+- **Editor** - Partial administrative access to the organization. Editors can create, edit, and delete repositories. They can also edit an existing team's access permissions.
+- **Organization owner** - Full organization administrative access. Organization owners can manage organization repositories, teams, members, settings, and billing.
+- **Company owner** - In addition to the permissions of an organization owner, company owners can configure settings for their associated organizations.
+
+Owners can manage roles for members of an organization on [Docker Hub](/docker-hub/members/#update-a-member-role), as well members of an [organization](/admin/organization/members/#update-a-member-role) or a [company](/admin/company/users/#update-a-member-role) on [Docker Admin](/admin/).
+
+## Permissions
+
+The following sections outline the permissions for each role.
+
+### Content and registry permissions
+
+| Permission | Member | Editor | Organization owner | Company owner |
+|:----------------------- |:------ |:-------|:------------------ |:----------- |
+| Explore images and extensions | ✅ | ✅ | ✅ | ✅ |
+| Star, favorite, vote, and comment on content | ✅ | ✅ | ✅ | ✅ |
+| Pull images | ✅ | ✅ | ✅ | ✅ |
+| Create and publish an extension | ✅ | ✅ | ✅ | ✅ |
+| Become a Verified, Official, or Open Source publisher | ❌ | ❌ | ✅ | ✅ |
+| Observe content engagement as a publisher | ❌ | ❌ | ✅ | ✅ |
+| Create public and private repositories | ❌ | ✅ | ✅ | ✅ |
+| Push, edit, and delete a repository | ❌ | ✅ | ✅ | ✅ |
+| Manage tags | ❌ | ✅ | ✅ | ✅ |
+| View repository activity | ❌ | ❌ | ✅ | ✅ |
+| Set up Automated builds | ❌ | ❌ | ✅ | ❌ |
```

Comment:
Members can do this as well - repository permissions are team based, not org-role based.